<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Services</string>
  </property>
  <property name="styleSheet">
   <string>
     QMainWindow { background-color: #2b2b2b; }
     QWidget { background-color: #2b2b2b; color: #ffffff; }
     QTabWidget::pane { background: #3c3c3c; }
     QTabBar::tab { background-color: #3c3c3c; color: #ffffff; }
     QTabBar::tab:selected { background-color: #4d4d4d; }
     QLabel { color: #ffffff; }
     QLineEdit { background-color: #4d4d4d; color: #ffffff; border: 1px solid #6c6c6c; }
     QPushButton { background-color: #4d4d4d; color: #ffffff; border: 1px solid #6c6c6c; }
     QPushButton:hover { background-color: #5e5e5e; }
     QTableView { background-color: #3c3c3c; color: #ffffff; }
   </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>580</width>
      <height>450</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabNouvelService">
     <attribute name="title">
      <string>Nouvel Service</string>
     </attribute>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_description">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_description">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_prix">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_duree">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Durée</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_duree">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_equipement">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Équipement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_equipement">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_employee">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Employee</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_employee">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>310</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabConsulterListe">
     <attribute name="title">
      <string>Consulter la liste</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>560</width>
        <height>340</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_extract">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Extract</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabEliminerService">
     <attribute name="title">
      <string>Éliminer un Service</string>
     </attribute>
     <widget class="QLabel" name="label_id_delete">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_delete">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabStatistics">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QLabel" name="label_statisticsTitle">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Service Statistics</string>
      </property>
     </widget>
     <widget class="QFrame" name="frameCharts">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>760</width>
        <height>200</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_charts">
       <item>
        <widget class="QLabel" name="label_chart2">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frameSummary">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>760</width>
        <height>200</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_summary">
       <item>
        <widget class="QLabel" name="label_totalServices">
         <property name="text">
          <string>Total Services: 123</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_totalRevenue">
         <property name="text">
          <string>Total Revenue: $12,345</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_averageServiceTime">
         <property name="text">
          <string>Average Service Time: 45 mins</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
